### ISSUE TITLE
LXC containers. Anyone interested?

### ISSUE THREAD

**Issue by KipourosV** [htpcBeginner/AtoMiC-ToolKit#285]
I've been using this (AtoMiC-ToolKit) amazing script to deploy it's apps/services in LXC containers inside a Proxmox host.
(one service/app per LXC container)

I was thinking that maybe we could modify the script, to accommodate this situation.
What I mean is, create a modified version of the script that can be run on a proxmox/LXC host
and instead of installing the service/app locally on the host, the script will deploy a container
and automatically do it's magic inside the container...

I'm in the proccess of doing some research on the matter and will post more as I find.

I am willing to actively help/work on this and will continue sharing my work/findings here.

Does anyone else find this idea interesting?

**Comment by postmanpaul** [comment]
I whould love to see this happen.. specal, i´m working on proxmox :)

**Comment by KipourosV** [comment]
Ok I'm doing some reaserch on pct, the command that manages containers in proxmox...

**Comment by copcopcopcop** [comment]
I would be interested

**Comment by Taymurf** [comment]
I would be interested, running this awesome script in a bunch of LXC containers the same way as OP.

**Comment by nemchik** [comment]
This is getting a decent bit of attention so I wanted to acknowledge it and comment that an announcement has been made in the readme https://github.com/htpcBeginner/AtoMiC-ToolKit/blob/master/README.md

**Comment by KipourosV** [comment]
Can't see any reference on LXC. Only Docker...

**Comment by nemchik** [comment]
Yeah I'm still looking into whether LXC would be a fit for the project.

**Comment by KipourosV** [comment]
Well we could fork? (I'm not very familiar with the git process).

**Comment by josephpyke1987** [comment]
I would be happy to help out and test a lxc version

**Comment by KipourosV** [comment]
There is an easy way to do this...
After creating your container manually,
you can pct exec and run the installation of atk command (apt install git, git clone atk, adduser etc) and then run atk --install appname... Hmm...

**Comment by postmanpaul** [comment]
Just tell me what ya need.. and ill do my best!! U needto config the space and things like that.. im gona goolge cmds for it after this weekend. Start on monday

**Comment by postmanpaul** [comment]
http://download.proxmox.com/images/system/  looking for this?

**Comment by postmanpaul** [comment]
https://forum.proxmox.com/threads/ubuntu-16-04-template-for-proxmox-4-1.27034/

**Comment by nemchik** [comment]
Please keep in mind that ATK likely won't be receiving any more updates, so if you're using atk to build your containers and you have a problem or want a new app that's not in ATK you'll be on your own.

You can fork the repo and maintain it if you'd like, or even make pull requests to get the fixes into this repository.

**Comment by postmanpaul** [comment]
Is atk gona quit?

**Comment by nemchik** [comment]
As of right now there is no one actively developing ATK. That doesn't mean ATK is dead, someone else could take over. Just be aware that the creator and others who have managed ATK have moved on to docker and new projects as indicated in the readme.
